src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -27,17 +41,19 @@ function SidebarMenuItem({ children, label, path, activePaths, onClick }) {
     <>
       <Link
         href={getPathWithSoruce(path)}
-        as={SectionLink}
         id={path}
-        isActive={isActive}
+        className={cn(
+          styles.sectionLink,
+          isActive && styles.active,
+          isRootParent && 'docSearch-lvl0'
+        )}
         onClick={onClick}
-        className={isRootParent ? 'docSearch-lvl0' : ''}
       >
         {label}
       </Link>
       {children && (
-        <Collapse isOpened={isActive}>
-          {children.map(item => (
+        <Collapse isOpened={!!isActive}>
```

Comment:
? is there bool(something) in JS ?

Reply:
`Boolean(something)` or its shortcut `!!something`. They are equivalent